great_expectations/core/__init__.py

```diff
@@ -18,23 +18,25 @@
 from .id_dict import IDDict
 from .run_identifier import RunIdentifier, RunIdentifierSchema
 from .urn import ge_urn
+from .validation_config import ValidationConfig
```

Comment:
Name is temporary and will be updated based on team consensus in the near future